charts/gateway/templates/tests/test-nodeport.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "trino-gateway.fullname" . }}-test-connection"
```

Comment:
Why this is a separate test? If it does the same as the existing `test-connection`, just using a different URL, it could be a single test.